sa/proto/sa.proto

```diff
@@ -125,6 +125,7 @@ message CountCertificatesByNamesRequest {
 
 message CountByNames {
   map<string, int64> counts = 1;
+  int64 earliest = 2; // Unix timestamp (nanoseconds)
```

Comment:
This is a new field! A perfect opportunity to use `google.protobuf.Timestamp` instead of messy int64s. I think this may also help get rid of the time-pointers and nil checks.

Reply:
Okay, adding it was also an excuse to play with the new shiny type 😅